tests/scorers/test_bias_scorer.py

```diff
@@ -17,14 +17,14 @@ def weave_bias_scorer():
 def test_bias_scorer_simple(weave_bias_scorer):
     """Tests that the scorer can handle a basic string."""
     output = "This is a balanced statement with no bias."
-    result = weave_bias_scorer.score(output)
-    assert not result.passed
+    result = weave_bias_scorer.score(output=output)
+    assert result.passed
```

Comment:
Why is this the opposite now?

Reply:
looks like it was previously using one of the TINY models, these are the same architecture but have most of their middle layers taken out so they're lightweight. I don't think we should have been using these for tests where we're asserting that the model evaluates some text correctly.

I've updated the model to use the full scorer, which will correctly return `result.passed == True`